scripts/generateDisclosure.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+const fs = require('fs')
+const generateDisclosurePdf = require('../server/meca/file-generators/disclosure')
+const { Manuscript } = require('@elifesciences/xpub-model')
+const { db } = require('pubsweet-server')
+
+async function regenerate(id) {
+  console.log(id)
+
+  const [manuscript] = await Manuscript.query().where({
+    'manuscript.id': id,
+  })
+
+  let doc = null
+  try {
+    await manuscript.$loadRelated('[teams]')
+    doc = await generateDisclosurePdf(manuscript, 'no-ip-address')
+
+    const filename = `disclosure-${id}.pdf`
+    fs.writeFile(filename, doc, err => {
+      if (err) {
+        return console.error(err)
+      }
+      console.log(`Written: ${filename}`)
+      return true
+    })
+  } catch (err) {
+    console.error(err)
+  } finally {
+    // we need to do this whenever we use the xpub-model in order that the
+    // process cleanly exits.
+    db.destroy()
+  }
+}
+
+// Run from the command line takes a manuscript id for the first arg.
+if (process.argv.length < 3 || !process.argv[2]) {
+  console.error('Need to supply a manuscript id as an argument!')
+  process.exit(1)
+}
+
+const id = process.argv[2]
+
+if (id.length !== 36) {
+  console.error('That does not look like a manuscript id')
+  process.exit(1)
+}
+
+regenerate(id)
```

Comment:
What's the need for this script?

Reply:
I assume this script is to be executed on a production instance.

Reply:
If not, we can :shipit: and test it afterwards.

Reply:
It's a manual running of the generation process to create a disclosure pdf for those that are missing it because of the bug. I agree this should have a test written for it to make sure it's doing what we assume it should.

Reply:
Every regenerated pdf will be eyeballed anyway so don't think tests are needed.